CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+* `GH`: Modifiation of any repository-related elements (screenshots, readme, etc.)
+* `MAPS`: Modification of `.map` files.
+* `MODELS`: Modification of models, both `.blend` and exported `.mdl`
+* `QC`: Modification of QuakeC source code.
+* `AUDIO`: Modification of game sound effects or music.
+* `WADS`: Modification of **map** textures, to be packed into a `.wad` during the build process.
+
+Here is a good example for a commit subject line:
+
+`GFX: Fix colormap having incorrect fullbright count`
+
+When more detail about a change is necessary, please take the opportunity to write a more detailed description in the commit body.
+
+If your commit addresses an open issue, be sure to include the issue in the commit message body.
+
+If you are committing on behalf of someone, please use the [Co-Author feature](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) if they other author has a GitHub profile. If they do not, please credit them in the commit body at the very bottom of the message as to keep the flow of the commit organized. You may use any source platform of their choice to refer to them as long as it complies with GitHub's Terms of Service.
+
+## Pull Requests
+
+Please keep Pull Requests on-topic. The Pull Request title should follow the same naming convention as your commit messages for clarity.
+
+It is required that you provide both a detailed desctription of your changes as well as visualization of your changes being functional in-game. The Pull Request template will guide you through this.
+
+Be mindful that Pull Requests are scrutinized and reviewed carefully to mitigate any potential problems a Collaborator may discover. It is important for us to verify and test your changes independently as well. We will leave reviews and mark appropriately when we request changes and are happy to turn the request into a discussion.
+
+Please do not have discussions about a Pull Request off-platform. We should preserve the history of the conversation in an easily accessible matter so that justifications can be tracked in the event of an overturning discussion.
+
+When having discussions, please be mindful to use proper grammar and full sentences in the English language. We should be accessible to people who are English-Second-Language, and a big part of that is refraining from using acronyms or shorthand/slang.
+
+If you are a Collaborator looking to merge a Pull Request, or are just curious about merge prerequisites, we will merge a Pull Request given the following are true:
+
+* It has received 2 or more approvals.
+* The Pull Request template has been followed.
+* Commits follow our convention standards.
+* If the user is a collaborator, the Branch name convention standards have been followed.
+* Our CI passes without failure.
+* 48 hours or more have passed since the PR has been marked ready for review.
```

Comment:
I see the intention here but I disagree with adding a 48h delay for everything but critical issues. Sometimes development just wants to move forward. If there are 4 approvals on a PR and people want something to go in it should be able to. Like if we could move "We really want it" under the banner of "critical" then I'm okay with this.

Reply:
Would you be alright if this decreased to 24 hours? We have had several occurrences wherein we have rushed PRs, with multiple approvals, and paid the price later where we realize there was a case we forgot to address (I have been one to make these sorts of mistakes as well)

Some of us work full time and that can be detrimental to our ability to review a PR as soon as it is opened. My worry is that without this getting we will be less able to catch issues before they make it into prod because I or someone else wasnt around in that 6 hour window, or whatever it may be.

Reply:
Yeah I think I would like 24h better. As I said I do understand the concern.

Reply:
maybe 48 with 2 reviews and 24 for 3 reviews?